pkg/vdri/trustbloc/vdri.go

```diff
@@ -61,11 +61,14 @@ type VDRI struct {
 	authToken        string
 
 	validatedConsortium map[string]bool
+
+	enableSignatureVerification bool
 }
 
 // New creates new bloc vdri
 func New(opts ...Option) *VDRI {
 	v := &VDRI{}
+	v.enableSignatureVerification = false
```

Comment:
default for bool is false so you don't need to set it

Reply:
Wanted it to be explicit, that false is the intended default

Reply:
(rather than an oversight that someone might think to fix in the future)

Reply:
it creates noise in the code, not sure who that someone could be ...